src/test/java/com/facebook/openwifirrm/optimizers/tpc/RandomTxPowerInitializerTest.java

```diff
@@ -23,39 +22,76 @@
 import com.facebook.openwifirrm.ucentral.UCentralConstants;
 import com.facebook.openwifirrm.ucentral.models.State;
 
+import static org.junit.jupiter.api.Assertions.*;
+
 @TestMethodOrder(OrderAnnotation.class)
 public class RandomTxPowerInitializerTest {
 	/** Test zone name. */
 	private static final String TEST_ZONE = "test-zone";
 
+	/** Serial numbers */
```

Comment:
nit: use a regular non-Javadoc comment for this (`//`) because Javadoc only applies to the *first* variable and not the second, which is misleading

Reply:
^

Reply:
Do you want me to add java doc for both two serial numbers?

Reply:
Either both or neither (I'd do neither since it's just a test class, but whatever you want)